Add author and talk info for workshop - 202308 - Aldrin

drin opened the pull request:
This adds bio info, an avatar image, and some initial info for the "skytether" talk.

The talk will probably be named something like "Query processing for a computational storage system" and will focus more on work this summer and how it leads into work in the Fall, but at a high enough level to discuss the value of processing queries for distributed execution.

This also updates the agenda with appropriate links to the bio and talk info.

drin committed:
task: Add author info for Aldrin

drin committed:
minor: fix whitespace

checking if newlines were inserted in the text instead of soft wraps

drin committed:
minor: removed whitespace

removing newlines (which shouldn't be there anyways because of soft wrapping?)

drin committed:
task: add photo for Aldrin

drin committed:
minor: renamed avatar image

drin committed:
task: update info for Aldrin's talk

carlosmalt commented on content/authors/aldrin.montana/_index.md:
```diff
@@ -0,0 +1,57 @@
+role: PhD Candidate
+
+# Organizations/Affiliations
+organizations: UC Santa Cruz
```
Build failed because `organizations:` requires a list of `name:`/`url:` pairs. For example:
```
organizations:
- name: Center for Computing Research (CCR)
  url: https://www.sandia.gov/ccr/
- name: Sandia National Laboratories
  url: https://www.sandia.gov/
```

drin replied:
thanks! I couldn't decipher the error message

carlosmalt reviewed (changes requested):
Thanks, @drin -- I'm going to wait until you have fixed the `organizations:` value and added an abstract.

carlosmalt commented:
... and I just noticed that this is a draft PR. Sorry!

drin committed:
fix: updated organization

Apparently `organizations` is a structured field rather than a text field

carlosmalt reviewed (approved):
Thanks @drin !

drin commented:
lol still a draft! I need to update the abstract.

carlosmalt commented:
Hey @drin -- In #20 I've separated profiles from talk abstracts. I removed titles and abstracts from profiles and moved them to `/content/talks/20230817`. I also updated `agenda.csv` with the new abstract links -- which is probably why you are seeing conflicts.

drin commented:
no problem, I can rebase

drin commented:
rebased and cherry-picked 6b54e728b8ab0d4a13624d533bdc3198f146ddde

drin committed:
task: added abstract and moved to a separate file

drin committed:
task: updated reference to Aldrin's talk

drin committed:
task: fixed reference to talk

apparently the path to the talk info uses a '.' where the path to the
author info uses a '-'?

drin commented:
marked ready for review.

drin committed:
minor: updated end of talk abstract
